include/globals.h

```diff
@@ -0,0 +1,16 @@
+#ifndef __GLOBALS_H_
```

Comment:
Here and elsewhere, ifdef guards must never start with __, as that’s reserved for the compiler. I suggest `KASSE_GLOBALS_H_` instead.

Reply:
Ah, good to know. My Emacs config auto-generated this bit of code.
I guess I should go with the convention we have in the other source files: _GLOBAL_H_